setup.cfg

```diff
@@ -26,6 +25,9 @@ classifiers=
 [options]
 packages = find:
 python_requires = >= 3.7
+setup_requires =
+  setuptools>=45.0
```

Comment:
Setup_requires is pretty much broken, it's strongly recommended not to add it to anything new